Uchu.World/Client/CdClient/TableCache.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Uchu.Core.Client;
+
+namespace Uchu.World.Client
+{
+    /// <summary>
+    /// Temporarily caches data for when a lot of client database
+    /// reads are done at once, such as when the world starts.
+    /// </summary>
+    public class TableCache<T> where T : class
+    {
+        private PropertyInfo cdClientContextProperty;
```

Comment:
Use uppercase properties here, lowercase are reserved for local variables, for example: CdClientContextProperty and CachedData.

Reply:
Resolved with the latest commit.